Makefile

```diff
@@ -3,13 +3,16 @@
 PY_FILES = $(strip $(shell find services -iname '*.py'))
 
 build:
+	docker-compose -f services/docker-compose.yml build
```

Comment:
My proposal would be 3 flavors:
- dev (for development)
- test or production (for testing in production environment)
- deploy (for the swarm)
then only 2 recipes:
- up (calls docker swarm init, docker-compose ... up --build)
- down (as of now)